ClipboardMonitor/NativeExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Ignore Spelling: Pwstr
+
+using System;
+using Windows.Win32.Foundation;
+
+namespace ClipboardMonitor
+{
+    internal static class NativeExtensions
+    {
+        public static HWND AsHwnd(this IntPtr intPtr) => new(intPtr);
```

Comment:
FWIW, the CsWin32-generated `HWND` type includes an explicit conversion operator for `IntPtr` already. So you *could* remove this and just do this:

```cs
IntPtr v = /*something*/;
HWND h = (HWND)v;
```

I would advise against defining new methods where a conversion operator already exists.